macros/AddonSidebar.ejs

```diff
@@ -278,89 +278,102 @@ var text = mdn.localStringMap({
-        <li><a href="<%=baseURL%>WebExtensions/Temporary_Installation_in_Firefox"><%=text['WebExtensions/Temporary_Installation_in_Firefox']%></a></li>
-        <li><a href="<%=baseURL%>WebExtensions/Debugging"><%=text['WebExtensions/Debugging']%></a></li>
-        <li><a href="<%=baseURL%>WebExtensions/Testing_persistent_and_restart_features"><%=text['WebExtensions/Testing_persistent_and_restart_features']%></a></li>
-        <li><a href="<%=baseURL%>WebExtensions/Developing_WebExtensions_for_Firefox_for_Android"><%=text['WebExtensions/Developing_WebExtensions_for_Firefox_for_Android']%></a></li>
-        <li><a href="<%=baseURL%>WebExtensions/Getting_started_with_web-ext"><%=text['WebExtensions/Getting_started_with_web-ext']%></a></li>
-        <li><a href="<%=baseURL%>WebExtensions/web-ext_command_reference"><%=text['WebExtensions/web-ext_command_reference']%></a></li>
-        <li><a href="<%=baseURL%>WebExtensions/WebExtensions_and_the_Add-on_ID"><%=text['WebExtensions/WebExtensions_and_the_Add-on_ID']%></a></li>
-        <li><a href="<%=baseURL%>WebExtensions/Request_the_right_permissions"><%=text['WebExtensions/Request_the_right_permissions']%></a></li>
-
-      </ol>
+    <li class="toggle">
+      <details>
+        <summary><%=text['WebExtensions#Firefox_workflow']%></summary>
+        <ol>
+          <li><a href="<%=baseURL%>WebExtensions/User_experience_best_practices"><%=text['WebExtensions/User_experience_best_practices']%></a></li>
+          <li><a href="<%=baseURL%>WebExtensions/Temporary_Installation_in_Firefox"><%=text['WebExtensions/Temporary_Installation_in_Firefox']%></a></li>
+          <li><a href="<%=baseURL%>WebExtensions/Debugging"><%=text['WebExtensions/Debugging']%></a></li>
+          <li><a href="<%=baseURL%>WebExtensions/Testing_persistent_and_restart_features"><%=text['WebExtensions/Testing_persistent_and_restart_features']%></a></li>
+          <li><a href="<%=baseURL%>WebExtensions/Developing_WebExtensions_for_Firefox_for_Android"><%=text['WebExtensions/Developing_WebExtensions_for_Firefox_for_Android']%></a></li>
+          <li><a href="<%=baseURL%>WebExtensions/Getting_started_with_web-ext"><%=text['WebExtensions/Getting_started_with_web-ext']%></a></li>
+          <li><a href="<%=baseURL%>WebExtensions/web-ext_command_reference"><%=text['WebExtensions/web-ext_command_reference']%></a></li>
+          <li><a href="<%=baseURL%>WebExtensions/WebExtensions_and_the_Add-on_ID"><%=text['WebExtensions/WebExtensions_and_the_Add-on_ID']%></a></li>
+          <li><a href="<%=baseURL%>WebExtensions/Request_the_right_permissions"><%=text['WebExtensions/Request_the_right_permissions']%></a></li>
+        </ol>
+      </details>
     </li>
-
-    <li data-default-state="<%=currentPageIsUnder('WebExtensions/API')%>"><a href="<%=baseURL%>WebExtensions/API"><%=text['WebExtensions/API']%></a>
-     <%-template("WebExtAPISidebar", [])%>
+    <li class="toggle">
+      <details <%=currentPageIsUnder('WebExtensions/API')%>>
```

Comment:
Nit: GitHub hates ``%>>``. I guess [EJS](https://github.com/github/linguist/blob/8bf9efa3702a1a43df85dc5cd72b63f3ff36871f/lib/linguist/languages.yml#L1133-L1140) isn't popular enough for a robust syntax highlighter? 😉 

The ["Whitespace Slurping" ending tag](http://www.ejs.co/) may help appease GitHub and get the desired output:

```ejs
<details <%=currentPageIsUnder('WebExtensions/API')_%> >
```

Nope, that seems worse. Maybe it doesn't like HTML escaping into an unquoted context?

```EJS
<details <%- currentPageIsUnder('WebExtensions/API')_%> >
```

Nope, syntax highlighting just seems to hate us. No change needed.

Reply:
All EJS (and Liquid/Jinja2/&lt;insert templating language of choice here&gt;) syntax highlighters I tried can’t properly highlight conditional attributes,